.github/workflows/cypress.yml

```diff
@@ -15,11 +15,11 @@ jobs:
         runs-on: ubuntu-latest
         steps:
             - name: Checkout code
-              uses: actions/checkout@v3
+              uses: actions/checkout@v4
             # Install yarn dependencies, cache them correctly
             # and run all Cypress tests
             - name: Cypress run
-              uses: cypress-io/github-action@v5.6.1
```

Comment:
not part of this atomic PR context, but the refactor is in the same code space so approved by me! 😸